.github/workflows/ci.yml

```diff
@@ -79,3 +83,7 @@ jobs:
           token: ${{ secrets.DENOBOT_PAT }}
           git-user-name: denobot
           git-user-email: denobot@users.noreply.github.com
```

Comment:
Let's remove this publish to /x and get tag version.

Reply:
Done